setup.py

```diff
@@ -77,23 +77,30 @@ def is_platform_mac():
     'long_description': readme,
     'long_description_content_type': 'text/markdown',
     'package_dir': {'pyuvdata': 'pyuvdata'},
-    'packages': ['pyuvdata', 'pyuvdata.tests'],
+    'packages': [
+        'pyuvdata',
+        'pyuvdata.tests',
+        'pyuvdata.uvbeam',
+        'pyuvdata.uvcal',
+        'pyuvdata.uvdata',
+        'pyuvdata.uvflag',
```

Comment:
Do we explicitly need this? should check on this.

Reply:
seems like it made no difference removing these. We can watch it and patch in something if there are issues.